test/groovy/ArtifactSetVersionTest.groovy

```diff
@@ -75,9 +77,8 @@ class ArtifactSetVersionTest extends BasePiperTest {
         assertThat(jscr.shell, hasItem("mvn --file 'pom.xml' --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn versions:set -DnewVersion=1.2.3-20180101010203_testCommitId"))
         assertThat(jscr.shell, hasItem('git add .'))
         assertThat(jscr.shell, hasItem("git commit -m 'update version 1.2.3-20180101010203_testCommitId'"))
-        assertThat(jscr.shell, hasItem("git remote set-url origin myGitSshUrl"))
-        assertThat(jscr.shell, hasItem("git tag build_1.2.3-20180101010203_testCommitId"))
-        assertThat(jscr.shell, hasItem("git push origin build_1.2.3-20180101010203_testCommitId"))
+        assertThat(jscr.shell.find{ e -> e.contains("git tag build_1.2.3-20180101010203_testCommitId")}, notNullValue())
```

Comment:
use `containsString` matcher

Reply:
containsString does not work here. jscr.shell is not a string, but a list. contains string looks for sub strings in strings. We are looking for one of the list entries containing a substring.

Reply:
yes, thats better. It fits more in the existing habits. Changed.